source/mainnet/net/guides/become-baker.rst

```diff
@@ -232,36 +223,34 @@ chain as soon as the transaction is included in a block, it can be consulted thr
 
    ...
 
-.. table:: Timeline: decreasing the stake
-
-   +----------------------------------------+-----------------------------------------+----------------------------------------+
-   |                                        | When transaction is included in a block | After *2 + bakerCooldownEpochs* epochs |
-   +========================================+=========================================+========================================+
-   | Change is visible by querying the node | ✓                                       |                                        |
-   +----------------------------------------+-----------------------------------------+----------------------------------------+
-   | Baker uses the new stake               |                                         | ✓                                      |
-   +----------------------------------------+-----------------------------------------+----------------------------------------+
-   | Stake can be decreased again or        | ✗                                       | ✓                                      |
-   | baker can be removed                   |                                         |                                        |
-   +----------------------------------------+-----------------------------------------+----------------------------------------+
++------------------------------------------+-----------------------------------------+-------------------------------+
+|                                          | When transaction is included in a block | First pay day after cool-down |
++==========================================+=========================================+===============================+
+| Change is visible by querying the node   | ✓                                       |                               |
++------------------------------------------+-----------------------------------------+-------------------------------+
+| Baker uses the new stake                 |                                         | ✓                             |
++------------------------------------------+-----------------------------------------+-------------------------------+
+| Stake can be decreased again or baker    | ✗                                       |                               |
```

Comment:
There should be a check mark in decreasing after cool-down. I just forgot to add it in my battle to get the tables to look correct.
The increasing table is the table above this one.

Reply:
Ah, OK. But then I still don't get why there is an x in the first column, whereas in other rows, there are just empty spaces. Is there a difference in meaning?

The table above is about the timing of what happens when you increase the stake. What I think is missing is the following: When I decrease my stake (i.e., I'm in this table), at what point in time can I decide to increase the stake again?

Reply:
The X has always been there, but I've removed it now because there shouldn't be a difference between an X or blank. I've added increase to the last line of that table as well since you can't do anything to the stake when you're in cool-down.